index.ts

```diff
@@ -0,0 +1 @@
+console.log('You helplessly stare into the void as you try to gain your bearings. Your foot catches on what you believe to be a loose flagstone. You fall and break your neck. You are dead.');
```

Comment:
Is this from https://github.com/jshemas/textAdventure/blob/gh-pages/adventure.json ?